### PR TITLE
Crypto Transfer Component - UI

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### CODE REVIEW

File: SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/compose/CryptoTransferView_Content.kt [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                lineHeight = 24.sp,
+                fontFamily = FontFamily(Font(R.font.inter_regular)),
+                fontWeight = FontWeight(400),
+                color = Color(0xDE000000),
+                letterSpacing = 0.5.sp,
+            )
+        )
+
+        // -- Max Button
+        if (!isTransferInFiat.value) {
```
Comment (brent-cybrid) [inline SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/compose/CryptoTransferView_Content.kt:238]: Why are we referencing fiat on a crypto transfer component?

File: SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/modal/compose/CryptoTransferModal_Quote.kt [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        ) + " ${cryptoTransferViewModel.fiat.code}"
+        CryptoTransferModal_Quote__Item(
+            modifier = Modifier
+                .constrainAs(fee) {
+                    start.linkTo(parent.start, margin = horizontalMargin)
+                    top.linkTo(amount.bottom, margin = 25.dp)
+                    end.linkTo(parent.end, margin = horizontalMargin)
+                    width = Dimension.fillToConstraints
+                },
+            titleText = "Transaction fee",
```
Comment (brent-cybrid) [inline SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/modal/compose/CryptoTransferModal_Quote.kt]: Localized text

**Review by brent-cybrid — COMMENTED** [review]
Component needs to be localized

File: SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/compose/CryptoTransferView_Content.kt (hunk 1 above)
Comment (nalancer08) [reply SDKAndroid/src/main/kotlin/app/cybrid/sdkandroid/components/cryptoTransfer/compose/CryptoTransferView_Content.kt:238]: Sorry bad name, I changed to `isAmountInFiat`

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**